### ISSUE TITLE
MAGIC dl3 file contains `OBS_MODE` twice with different values:

### ISSUE THREAD

**Issue by maxnoe** [gammapy/gammapy-data#30]
File `GAMMAPY_DATA/magic/rad_max/data/20131004_05029747_DL3_CrabNebula-W0.40+035.fits` contains the OBS_MODE keyword twice, once with `POINTING`, once with `Wobble` content resulting in astropy giving a list of values when accessing it via `Table.meta`

**Comment by maxnoe** [comment]
```
In [4]: hdul = fits.open("GAMMAPY_DATA/magic/rad_max/data/20131004_05029747_DL3_CrabNebula-W0.40+035.fits")

In [5]: hdul["EVENTS"].header
Out[5]: 
XTENSION= 'BINTABLE'           / binary table extension                         
BITPIX  =                    8 / 8-bit bytes                                    
NAXIS   =                    2 / 2-dimensional binary table                     
NAXIS1  =                   28 / width of table in bytes                        
NAXIS2  =                11189 / number of rows in table                        
PCOUNT  =                    0 / size of special data area                      
GCOUNT  =                    1 / one data group (required keyword)              
TFIELDS =                    5 / number of fields in each row                   
TTYPE1  = 'EVENT_ID'           / label for field   1                            
TFORM1  = '1K      '           / data format of field: 8-byte INTEGER           
TTYPE2  = 'TIME    '           / label for field   2                            
TFORM2  = '1D      '           / data format of field: 8-byte DOUBLE            
TUNIT2  = 's       '           / physical unit of field                         
TTYPE3  = 'RA      '           / label for field   3                            
TFORM3  = '1E      '           / data format of field: 4-byte REAL              
TUNIT3  = 'deg     '           / physical unit of field                         
TTYPE4  = 'DEC     '           / label for field   4                            
TFORM4  = '1E      '           / data format of field: 4-byte REAL              
TUNIT4  = 'deg     '           / physical unit of field                         
TTYPE5  = 'ENERGY  '           / label for field   5                            
TFORM5  = '1E      '           / data format of field: 4-byte REAL              
TUNIT5  = 'TeV     '           / physical unit of field                         
EXTNAME = 'EVENTS  '           / name of this binary table extension            
HDUCLASS= 'GADF    '           / conformant with GADF specifications            
HDUDOC  = 'https://github.com/open-gamma-ray-astro/gamma-astro-data-formats' / d
HDUVERS = '0.2     '           / version of the format                          
HDUCLAS1= 'EVENTS  '           / primary extension class                        
OBS_ID  =              5029747 / unique observation identifier                  
TSTART  =         3.337788E+08 / [s] observation start time                     
TSTOP   =           3.3378E+08 / [s] observation end time                       
ONTIME  =                1188. / [s] total good time (sum of GTIs)              
LIVETIME=             1178.066 / [s] total effective time                       
DEADC   =            0.9916382 / dead time correction                           
OBS_MODE= 'POINTING'           / observation mode                               
EQUINOX =                2000. / equinox                                        
RADECSYS= 'FK5     '           / stellar reference frame                        
ORIGIN  = 'MAGIC Collaboration' / organisation that created the FITS file       
TELESCOP= 'MAGIC   '           / telescope                                      
INSTRUME= 'MAGIC stereo'       / instrument                                     
CREATOR = 'MAGIC_DL3'          / software that created the file                 
VERSION = '0.1.2   '           / version of the software that created the file  
MJDREFI =                52706 / [d] integer part of reference MJD              
MJDREFF =                   0. / [d] fractional part of reference MJD           
TIMEUNIT= 's       '           / time unit                                      
TIMESYS = 'UTC     '           / time system                                    
TIMEREF = 'local   '           / time reference frame                           
OBJECT  = 'CrabNebula'         / name of the source                             
RA_OBJ  =             83.63333 / [deg] source RA                                
DEC_OBJ =             22.01444 / [deg] source DEC                               
OBS_MODE= 'Wobble  '           / observation mode                               
RA_PNT  =             83.98333 / [deg] observation pointing RA                  
DEC_PNT =             22.24389 / [deg] observation pointing DEC                 
ALT_PNT =             69.69974 / [deg] average altitude pointing                
AZ_PNT  =             103.8848 / [deg] average azimuth pointing   
```

**Comment by cosimoNigro** [comment]
Hello @maxnoe, I think the one you are using is an older version of the file.
I fixed this problem in PR #27.

This is what I see downloading the file from git
```
XTENSION= 'BINTABLE'           / binary table extension                         
BITPIX  =                    8 / 8-bit bytes                                    
NAXIS   =                    2 / 2-dimensional binary table                     
NAXIS1  =                   28 / width of table in bytes                        
NAXIS2  =                11189 / number of rows in table                        
PCOUNT  =                    0 / size of special data area                      
GCOUNT  =                    1 / one data group (required keyword)              
TFIELDS =                    5 / number of fields in each row                   
TTYPE1  = 'EVENT_ID'           / label for field   1                            
TFORM1  = '1K      '           / data format of field: 8-byte INTEGER           
TTYPE2  = 'TIME    '           / label for field   2                            
TFORM2  = '1D      '           / data format of field: 8-byte DOUBLE            
TUNIT2  = 's       '           / physical unit of field                         
TTYPE3  = 'RA      '           / label for field   3                            
TFORM3  = '1E      '           / data format of field: 4-byte REAL              
TUNIT3  = 'deg     '           / physical unit of field                         
TTYPE4  = 'DEC     '           / label for field   4                            
TFORM4  = '1E      '           / data format of field: 4-byte REAL              
TUNIT4  = 'deg     '           / physical unit of field                         
TTYPE5  = 'ENERGY  '           / label for field   5                            
TFORM5  = '1E      '           / data format of field: 4-byte REAL              
TUNIT5  = 'TeV     '           / physical unit of field                         
EXTNAME = 'EVENTS  '           / name of this binary table extension            
HDUCLASS= 'GADF    '           / conformant with GADF specifications            
HDUDOC  = 'https://github.com/open-gamma-ray-astro/gamma-astro-data-formats' / d
HDUVERS = '0.2     '           / version of the format                          
HDUCLAS1= 'EVENTS  '           / primary extension class                        
OBS_ID  =              5029747 / unique observation identifier                  
TSTART  =     333778852.435217 / [s] observation start time                     
TSTOP   =     333780040.546979 / [s] observation end time                       
ONTIME  =       1188.111761868 / [s] total good time (sum of GTIs)              
LIVETIME=     1178.06621791733 / [s] total effective time                       
DEADC   =             0.991545 / dead time correction                           
EQUINOX =                2000. / equinox                                        
RADECSYS= 'FK5     '           / stellar reference frame                        
ORIGIN  = 'MAGIC Collaboration' / organisation that created the FITS file       
TELESCOP= 'MAGIC   '           / telescope                                      
INSTRUME= 'MAGIC stereo'       / instrument                                     
CREATOR = 'MAGIC_DL3'          / software that created the file                 
VERSION = '0.1.8   '           / version of the software that created the file  
MJDREFI =                52706 / [d] integer part of reference MJD              
MJDREFF =                   0. / [d] fractional part of reference MJD           
TIMEUNIT= 's       '           / time unit                                      
TIMESYS = 'UTC     '           / time system                                    
TIMEREF = 'local   '           / time reference frame                           
OBJECT  = 'CrabNebula'         / name of the source                             
RA_OBJ  =             83.63333 / [deg] source RA                                
DEC_OBJ =             22.01444 / [deg] source DEC                               
OBS_MODE= 'POINTING'           / observation mode                               
RA_PNT  =             83.98333 / [deg] observation pointing RA                  
DEC_PNT =             22.24389 / [deg] observation pointing DEC                 
GEOLON  =               -17.89 / [deg] Geographic longitude of array centre     
GEOLAT  =            28.761944 / [deg] Geographic latitude of array centre      
ALTITUDE=              2231.28 / [m] Altitude of array center above sea level   
ALT_PNT =             69.69974 / [deg] average altitude pointing                
AZ_PNT  =             103.8848 / [deg] average azimuth pointing
```

Let me know if updating to the latest version of the repo fixes it.

**Comment by maxnoe** [comment]
Hi, yes, redownloading the data fixed it, sorry!
